plugins/modules/ipagroup.py

```diff
@@ -272,6 +284,7 @@ def main():
             membermanager_user=dict(required=False, type='list', default=None),
             membermanager_group=dict(required=False, type='list',
                                      default=None),
+            external_member=dict(required=False, type='list', default=None),
```

Comment:
Please add the internal name as an alias `ipaexternalmember` - also in the docs.

Reply:
Fixed.